File: Ejercicio 10_Primer_Departamental.py
class Operaciones:

    def __init__(self):
        self.numeros = []

    def ingresar_numeros(self):
        i = 0
        while i < 5:
            num = float(input("Ingresa un numero: "))
            self.numeros.append(num)
            i += 1

    def calcular(self):
        suma = 0
        for n in self.numeros:
            suma += n

        media = suma / len(self.numeros)
        maximo = self.numeros[0]
        minimo = self.numeros[0]

        for n in self.numeros:
            if n > maximo:
                maximo = n
            if n < minimo:
                minimo = n

        print("Numeros ingresados:", self.numeros)
        print("Suma:", suma)
        print("Media:", media)
        print("Maximo:", maximo)
        print("Minimo:", minimo)


op = Operaciones()
op.ingresar_numeros()
op.calcular()

File: Ejercicio 10_Primer_Departamental.cpp
#include <iostream>

using namespace std;

int main() {
    const int N = 5;
    double numeros[N];
    double suma = 0;

    for (int i = 0; i < N; i++) {
        cout << "Ingresa numero " << i+1 << ": ";
        cin >> numeros[i];
        suma += numeros[i];
    }

    cout << "El promedio es: " << suma / N << endl;

    return 0;
}